source/windows/thread.c

```diff
@@ -14,29 +14,40 @@
 */
 
 #include <aws/common/thread.h>
+#include <assert.h>
+
+static struct aws_thread_options default_options = {
```

Comment:
Having a static mutable struct is just asking for someone to start modifying it from multiple threads.